src/core/card/LargeCard.tsx

```diff
@@ -0,0 +1,49 @@
+import { IconArrowRight } from 'hds-react'
+import React from 'react'
+// eslint-disable-next-line import/no-extraneous-dependencies
+import classNames from 'classnames'
+
+import styles from './largeCard.module.scss'
+
+export type LargeCardProps = {
+  imageUrl?: string
+  imagePosition?: 'right' | 'left'
+  subTitle?: string
+  text?: string
+  title?: string
+  url?: string
+}
+
+export default function LargeCard({
```

Comment:
After seeing an example in storybook I see they are used for different purposes, but if it could be cleanly done, I would merge the LargeCard to the Card with some kind of image-position-attribute or a variation-attribute.

Reply:
i might do it but later. in each and every project we have 2 different components